mentorship ios/Members/SupportingViews/SendRequestForm.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  SendRequestForm.swift
+//  mentorship ios
+//
+//  Created by Yugantar Jain on 10/06/20.
+//  Copyright © 2020 Yugantar Jain. All rights reserved.
+//
+
+import SwiftUI
+
+struct SendRequestForm: View {
+    var name: String
+    @Binding var pickerSelection: Int
+    @Binding var endDate: Date
+    @Binding var notesText: String
+    @Binding var notesFrame: CGRect
+    
+    var body: some View {
+        Form {
+            //heading
+            Section(header:
+                Text("To \(name)")
+                    .font(.title)
+                    .fontWeight(.bold)
+            ) {
+                EmptyView()
+            }
+            
+            //settings
+            Section {
+                VStack(alignment: .leading) {
+                    Text("My Role")
+                    
+                    Picker(selection: $pickerSelection, label: Text("My Role")) {
+                        Text("Mentee").tag(1)
+                        Text("Mentor").tag(2)
+                    }
+                    .pickerStyle(SegmentedPickerStyle())
+                    .labelsHidden()
+                }
+                
+                DatePicker(selection: $endDate, displayedComponents: .date) {
+                    Text("End Date")
+                }
+                
+                TextField("Notes", text: $notesText)
+                    .background(GeometryReader { geo -> Color in
+                        let frame = geo.frame(in: .global)
+                        print(frame)
+                        self.notesFrame = frame
```

Comment:
Please solve this error on line 50. (Modifying state during view update, this will cause undefined behavior.)
As this result in non-functionality of Send Request button after dismissing the SendRequestForm.

Reply:
Yes, I know. This is what I was talking about in yesterday's meeting, the only way I came across to add a custom button below a form. But I think we're going with the original design, so it's not an issue.

Reply:
ok then, if we are going with the old PR, please close this one!